dev/build-images

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env bash
+
+# Copyright 2022 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# script to build container images with go
+set -o errexit -o nounset -o pipefail
+
+# cd to the repo root
+REPO_ROOT=$(git rev-parse --show-toplevel)
+cd "${REPO_ROOT}"
+
+# overridable list of binaries to build images for
+IMAGES="${IMAGES:-cmd/redirectserver}"
+IFS=" " read -r -a images <<< "$IMAGES"
+# overridable registry to use
+KO_DOCKER_REPO="${KO_DOCKER_REPO:-gcr.io/k8s-staging-infra-tools}"
+export KO_DOCKER_REPO
+# push or local tar?
+PUSH="${PUSH:-false}"
+# overridable auto-tag
+TAG="${TAG:-"$(date +v%Y%m%d)-$(git describe --always --dirty)"}"
+
+mkdir -p bin/
+
+# build images
+# TODO: bake commit info into binaries consistently
+for image in "${images[@]}"; do
+    name="$(basename "${image}")"
+    # push or local tarball
+    publish_args=(--tarball=bin/"${name}".tar --push=false)
```

Comment:
```suggestion
    publish_args=(--tarball=bin/"${name}".tar --platform=linux/amd64,linux/arm64 --push=false)
```

Don't cost a lot to publish arm64 images. Maybe useful in the future if we want to to run on thing like Lambda Graviton3.

Reply:
Thanks - incorporated and marked you as a co-author!